client/src/App.js

```diff
@@ -1,13 +1,25 @@
 import React from "react";
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
 import Books from "./pages/Books";
 import Nav from "./components/Nav";
+import Users from "./pages/Books/Users";
+import Tasks from "./pages/Books/Tasks";
+import Comments from "./pages/Books/Comments";
 
 function App() {
   return (
-    <div>
```

Comment:
I think the div starting at the beginning should be closed at the end, that means the closing router has to be inside the </div>. Please correct me if i am wrong.

Reply:
I did get the impression that when using React Router that  that could be the main wrapper and function properly. Certainly if there isn't a matching closer we can clean that up. Oddly I did not experience any error when testing locally